sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs

```diff
@@ -78,12 +81,12 @@ private void SetBackoffState()
         {
             _state.Update(LiveMetricsState.Backoff);
             _shouldCollect = false;
-            _callbackAction = BackoffConcluded;
+            _callbackAction = OnBackoffConcluded;
```

Comment:
nit: Readability of code is a concern as these methods are across different files.

Reply:
`OnBackoffConcluded` is in this same file on line 89.

`OnPing` and `OnPost` are in Manager.RestClient because they're more involved than any of the methods in this class.
We can discuss different ways to organize this, but I propose we consider any more changes an enhancement that should be addressed after this PR.